composer.json

```diff
@@ -19,12 +19,13 @@
         "drupal-composer/drupal-scaffold": "~2.5.2",
         "drupal/config_devel": "~1.2",
         "drupal/drupal-extension": "~4.0",
+        "drupal/entity_browser": "^2.5",
```

Comment:
Why this new dev dependency?

Reply:
There is a patch that can't be applied on lower versions.

Reply:
Ok makes sense.